guidelines/index.html

```diff
@@ -78,1254 +106,1196 @@ <h3>Section status levels</h3>
-					<li><strong><a href="#pre-assessment-checks">Pre-assessment checks</a>:</strong> Pre-Assessment checks are tests or criteria that implementers can use to determine if they are ready to assess conformance. The intent is to help organizations prepare for conformance testing, not to create a new level of conformance. </li>
-				</ul>
-				<p>The details of these approaches change as we assemble them into a coherent whole.  This draft gives a high level overview of these approaches so we can give an update and receive feedback on the individual approaches we are considering.  </p> 
+				<section>
+					<h4>Pointer Focus Appearance</h4>
+					<p class="guideline-text">Users can see the location of the pointer focus.</p>
+					<section class="provision" data-status="exploratory">
+						<h5>Pointer visible</h5>
+						<p class="provision-text">There is a visible indicator of pointer focus.</p>
+					</section>
+				</section>
 
-			<div class="ednote">
-				<p>As we continue developing conformance, we seek input on the following:</p>
-				<ul>
-					<li>Which option has the best chance of adoption and why?</li>
-					<li>How well do these approaches support regulatory needs?</li>
-					<li>How will these approaches be integrated into a conformance model (including levels or scores)?</li>
-				</ul>
+				<section>
+					<h4>Navigating content</h4>
+					<p class="guideline-text">Users can determine where they are and move through content (including interactive elements) in a systematic and meaningful way regardless of input or movement method. </p>
+					<section class="provision" data-status="exploratory">
+						<h5>Focus in viewport</h5>
+						<p class="provision-text">The focus does not move to a position outside the current viewport, unless a mechanism is available to return to the previous focus point.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Focus retention</h5>
+						<p class="provision-text">A user can focus on a content “area,” such as a modal or pop-up, then resume their view of all content using a limited number of steps.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Keyboard focus order</h5>
+						<p class="provision-text">The keyboard focus moves sequentially through content in an order and way that preserves meaning and operability.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Restore focus</h5>
+						<p class="provision-text">When the focus is moved by the content into a temporary change of view (e.g. a modal), the focus is restored to its previous location when the view is restored.</p>
```

Comment:
Would it be easier to know where to move the focus if we reuse 'change of view' from the 'when'?

```suggestion
						<p class="provision-text">When the focus is moved by the content into a temporary change of view (e.g. a modal), the focus is restored to where it was before the change of view.</p>
```

Reply:
I see where you're going, but the timing aspect doesn't work. You are saying that when the temporary view _opens_, put the focus back to where it was before.

```suggestion
						<p class="provision-text">When the focus is moved by the content into a temporary change of view (e.g. a modal), the focus is restored to its previous location when the temporary change of view is removed.</p>
```

Reply:
Ah I see what you mean, your suggestion helps!